ironfish/src/wallet/wallet.ts

```diff
@@ -839,6 +839,16 @@ export class Wallet {
           continue
         }
 
+        // Skip expired transactions
+        if (
+          this.chain.verifier.isExpiredSequence(
+            transaction.expirationSequence(),
+            this.chain.head.sequence,
+          )
+        ) {
+          continue
+        }
+
```

Comment:
we should change this whole method to only loop over the expiring transactions store that was added.